geonode/base/api/tests.py

```diff
@@ -2658,6 +2661,123 @@ def test_only_get_method_is_available(self):
         response = self.client.put(url)
         self.assertEqual(response.status_code, 403)
 
+    def test_insert_one_linked_resource(self):
+        url = reverse("base-resources-linked_resources", args=[self.doc.id])
+
+        self.client.force_login(get_user_model().objects.get(username="admin"))
```

Comment:
MINOR: you can define `admin` in the `setUp` class like:
```python
self.admin = get_user_model().objects.get(username="admin")
```
and then call it in the test as `self.admin`